### PR TITLE
cult stun hand is unusable at halo stage

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

cult stun hand is unusable at halo stage
you may not carve this spell at halo stage and if you have it carved already and the cult gets Halos it is removed

## Why It's Good For The Game

right now any cultist at the halo stage tries to get a cheap stun in before meleeing you to death with their really big and fancy and good block chance sword
this is obviously not cool as this tends to shut down your opponent instantly and before they can even pick up their weapons and shields you can just cave their brain in with your sword which is just not good gameplay
with limiting the cult stun hand to the "stealth" (or the cult stages you should be stealthy and converting people) i believe this should resolve the issue, as if youre going loud with halos i dont believe you need to convert more people

## Changelog
:cl:
balance: cult stun hand is unusable at halo stage
/:cl:

### CODE REVIEW

**Comment by Metekillot** [comment]
Cult stun hand already has its timer nerfed into the ground at halo stage. Isn't it also blockable, and also hard-countered by holy melons and chap magick? Also; cult stun hand is subject to stun healing from all the drugs that heal immobility. Here are some permalinks.

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L90

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L178

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L457

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L91

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L604

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L873

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L1172

https://github.com/Metekillot/tgtekfork/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L1479

**Comment by EnterTheJake** [comment]
I think bombarding cult with nerfs is the wrong approach, they need a rework not being nerfed into uselessness.

**Comment by RedBaronFlyer** [comment]
Didn't cult already get a pretty significant stun hand nerf like, two weeks ago with how long the stun last?

**Comment by mc-oofert** [comment]
> I think bombarding cult with nerfs is the wrong approach, they need a rework not being nerfed into uselessness.

write a design doc and maybe your comment would hold some merit to it

**Comment by DaydreamIQ** [comment]
The stun currently only lasts about three seconds at halo stage compared to like...10 it was before? We've already got a serious problem of people choosing to murder over converting when halos pop so unless you got a solution to that as well don't make it worse

**Comment by EnterTheJake** [comment]
> > I think bombarding cult with nerfs is the wrong approach, they need a rework not being nerfed into uselessness.
> 
> write a rework design doc and maybe your comment would hold some merit to it

That's just arguing in bad faith my dude, i don't care about cult at all, but it doesn't take  a genious to realize this approach of just constantly nerfing the thing i dislike is just lazy and won't address the core issue.

If anything why don't you do it? this is not the first pr you've had on cult.

**Comment by EnterTheJake** [comment]
Also just for the record you can counter tier 3 stun with roundstart drugs.


https://github.com/tgstation/tgstation/assets/102721711/0dfda938-ab47-40fa-a392-ef61346eac51

**Comment by mc-oofert** [comment]
> Also just for the record you can counter tier 3 stun with roundstart drugs.
> cult_stun_hard_counter.mp4

still enough time to critically wound someone or just shove them but thankfully the latter should be getting sorted out by that shovestun pr

**Comment by Dorsisdwarf** [comment]
3 seconds still guarantees a kill with the big sword, doesnt it?

**Comment by EnterTheJake** [comment]
> > Also just for the record you can counter tier 3 stun with roundstart drugs.
> > cult_stun_hard_counter.mp4
> 
> still enough time to critically wound someone or just shove them but thankfully the latter should be getting sorted out by that shovestun pr

The stun showcased is the tier 1(no eyes,no halo), tier 2 and tier 3 stun gets purged instantly.

**Comment by mc-oofert** [comment]
> > > Also just for the record you can counter tier 3 stun with roundstart drugs.
> > > cult_stun_hard_counter.mp4
> > 
> > 
> > still enough time to critically wound someone or just shove them but thankfully the latter should be getting sorted out by that shovestun pr
> 
> The stun showcased is the tier 1(no eyes,no halo), tier 2 and tier 3 stun gets purged instantly.

you still get dropped

**Comment by GuillaumePrata** [comment]
The best part of removing the stun at halo is that it is the only consistent way to move the meta around.

Counters are fine in paper, but they have existed for years and nothing changes meta wise.

Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.

**Comment by RedBaronFlyer** [comment]
> The best part of removing the stun at halo is that it is the only consistent way to move the meta around.
> 
> Counters are fine in paper, but they have existed for years and nothing changes meta wise.
> 
> Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.

The issue is that people will default to the easiest option. Cult already has a massive issue of a huge chunk of the playerbase not knowing how to use spells or make armor/weapons/constructs. This will change absolutely nothing from there already being an issue of cultist killing people they should ideally be converting and instead make it to cult having an even bigger incentive to kill instead of converting at a certain point.

I don't like the idea of shifting "the meta" around if it means randomly shooting the kneecap of a pretty important tool. It'd be like if revolutionary flashes just stopped working on crewmembers past a certain point regardless that they didn't have mindshields. If anything cult stuff should be getting even more powerful the more cultist there are (which I don't agree with being the case by the way)

**Comment by the-orange-cow** [comment]
It is my understanding that the main point of the hand stun is to make converting people discreetly feasible.  Once halos are out, discreetness is pretty much out the window, so why would it need to retain that functionality?

File: code/__DEFINES/antagonists.dm [hunk 1]
```diff
@@ -407,3 +407,12 @@ GLOBAL_LIST_INIT(human_invader_antagonists, list(
 
 /// Camera net used by battle royale objective
 #define BATTLE_ROYALE_CAMERA_NET "battle_royale_camera_net"
+
+/// If this flag is present, the cult has reached the halo stage
+#define CULTSTATE_HALO (1 << 1)
```
Comment (Mothblocks) [inline code/__DEFINES/antagonists.dm:412]: nit: cult_state

**Comment by Ilgradhr** [comment]
> It is my understanding that the main point of the hand stun is to make converting people discreetly feasible. Once halos are out, discreetness is pretty much out the window, so why would it need to retain that functionality?

In most cases cult has only close combat equipment and crew usually has lasers, cargo and melons. I do not think that cult is overpowered cuz only few people know how to fight as a cult member against armed secoff or a tider with gear.

**Comment by Deultus** [comment]
> The best part of removing the stun at halo is that it is the only consistent way to move the meta around.
> 
> Counters are fine in paper, but they have existed for years and nothing changes meta wise.
> 
> Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.

there are a lot of funny things on paper, there are a lot of runes and spells, but there is no point in using them, they are either ineffective or require the participation of several people, which negates everything

**Comment by Deultus** [comment]
and the question also arises, but how to convert people, it remains to kill everyone you meet after the halo

**Comment by GuillaumePrata** [comment]
> In most cases cult has only close combat equipment

They have pocket auto rifles, just not used often because of how efficient 1 click win slaps are.

>  there are a lot of funny things on paper, there are a lot of runes and spells, but there is no point in using them, they are either ineffective...

Cult toys are all overtuned, it is why 1 solo experienced cultists can, and often will, carry all the other cultists by themselves.

None of these others tools matter much when you can carry 4 "I win if I get close to you" runes, and most people don't know how insane the other tools are because in 95% of the times they died to a cultist it was the slap and kill strat.

The Halberd is a ranged stun that will win any 1v1 if it connects, they have a pocket autorifle that heals cultists instead of doing friendly fire, throwing multiple daggers at people is a sleeper strat, etc, etc.

**Comment by Ilgradhr** [comment]
> > In most cases cult has only close combat equipment
> 
> They have pocket auto rifles, just not used often because of how efficient 1 click win slaps are.
> 
> > there are a lot of funny things on paper, there are a lot of runes and spells, but there is no point in using them, they are either ineffective...
> 
> Cult toys are all overtuned, it is why 1 solo experienced cultists can, and often will, carry all the other cultists by themselves.
> 
> None of these others tools matter much when you can carry 4 "I win if I get close to you" runes, and most people don't know how insane the other tools are because in 95% of the times they died to a cultist it was the slap and kill strat.
> 
> The Halberd is a ranged stun that will win any 1v1 if it connects, they have a pocket autorifle that heals cultists instead of doing friendly fire, throwing multiple daggers at people is a sleeper strat, etc, etc.

Aye there are some things that cult have and I wont name them that can be overpowered. But Blood Bolt Barrage and Halberd use a lot of blood and you need to farm it to get either and that takes soome time. And you cant store the orb it just goes away using full blood cost if you "drop" it by any chance. 

Also you cant carve more than 2 stun runes on yourself so yeah not 4 "I win if I get close to you". And when they have halo the stun timer is quite short had an encounter with halo cult few days ago and a geared cultist thought that it was a good idea to try to convert me... he died using both stuns on me. Unfortunately halbert shatters when it does hit making it a costly one time use range stun... 

The last thing i want to mention tg and terry in particular are fast pased and more than a third of cult dont have shit when halos show up. If you take away a cut version of stun spell from them they cant even defend themselves from crew and unlike most antags they have big giveaway that they are bad in form of a halo. And if you dont have cannon fodder pros can only chip at the crew with no converts most likely.

**Comment by Singul0** [comment]
> The best part of removing the stun at halo is that it is the only consistent way to move the meta around.
> 
> Counters are fine in paper, but they have existed for years and nothing changes meta wise.
> 
> Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.

maybe if you want to try and force the meta to change, you gotta buff the other toys cult have, I have tried using the lesser known toys cult uses and it is shit, it's not just the funny cult hand being op, the powercreep of everything else has made any cult stuff except for constructs and red hand unviable

**Comment by CCC23** [comment]
> > The best part of removing the stun at halo is that it is the only consistent way to move the meta around.
> > Counters are fine in paper, but they have existed for years and nothing changes meta wise.
> > Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.
> 
> maybe if you want to try and force the meta to change, you gotta buff the other toys cult have, I have tried using the lesser known toys cult uses and it is shit, it's not just the funny cult hand being op, the powercreep of everything else has made any cult stuff except for constructs and red hand unviable
> 
> cult is pretty weak already and is only held up by the funny stun hand. try buffing them instead of just making the cult weak so that the meta can change

"weak" infinite printable eswords with the strongest flat armor in the game space proof with magic teleport spells as a conversion antag with built in emps and revival. This antag is literally every antag mixed into a grab bag of some of their strongest abilities.

**Comment by Singul0** [comment]
> "weak" infinite printable eswords with the strongest flat armor in the game space proof with magic teleport spells as a conversion antag with built in emps and revival. This antag is literally every antag mixed into a grab bag of some of their strongest abilities.

hey don't ask me. look at the winrate

**Comment by carlarctg** [comment]
ided and a bad one at that

cult is certainly over complicated but haphazard patches like these help nothing

**Comment by GuillaumePrata** [comment]
> cult is certainly over complicated but haphazard patches like these help nothing

Every time something is getting nerfed people try to bring this up, even when the PR is quite clear on how the changes work and the goal it is trying to achieve.

Haphazard feedback like these help nothing.

Cult is overly complicated, but this PR is not trying to fix that, and we shouldn't lock ourselves out of balancing cult unless "It is fixing it's complicated bloat issues" or we end up like Genetics/Xenobio/Virology with almost no balance changes for years.

**Comment by Formerxboxgamer** [comment]
Bro if by halos you both: do not have holy melons and are still fighting cult in maint tunnels where melee matters you deserve to be fuumajinned.

**Comment by LSommers** [comment]
> > > The best part of removing the stun at halo is that it is the only consistent way to move the meta around.
> > > Counters are fine in paper, but they have existed for years and nothing changes meta wise.
> > > Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.
> > 
> > 
> > maybe if you want to try and force the meta to change, you gotta buff the other toys cult have, I have tried using the lesser known toys cult uses and it is shit, it's not just the funny cult hand being op, the powercreep of everything else has made any cult stuff except for constructs and red hand unviable
> > cult is pretty weak already and is only held up by the funny stun hand. try buffing them instead of just making the cult weak so that the meta can change
> 
> "weak" infinite printable eswords with the strongest flat armor in the game space proof with magic teleport spells as a conversion antag with built in emps and revival. This antag is literally every antag mixed into a grab bag of some of their strongest abilities.

The thing about all of these abilities that Cult has is that they rely on one quality that 90% of people don't grasp, which utterly hamstrings the cult:

Cooperation and communication.
Do you have any idea how many times I've watched a cult fumble on a summon rune because a significant chunk of the cult is too busy trying to murderbone and somehow can't see the GIANT RED TEXT from cult chat yelling "GO TO X NOW GO TO X NOW GO GO GO GO NOW NOW NOW X X NOW GO X NOW GO"?

Or how many times I've seen someone who actually gets runes trying to get anyone to stand next to it long enough to activate it? I don't think I've seen anyone pop a blood boil in years, it's sad. It's no wonder POCKET SHADE was such a meta strat for the longest time, it was the only way to get some of these clueless fuckwits to actually stand near a rune when you needed them.

**Comment by alien3301** [comment]
> > > > The best part of removing the stun at halo is that it is the only consistent way to move the meta around.
> > > > Counters are fine in paper, but they have existed for years and nothing changes meta wise.
> > > > Cult has a lot of fun toys that no one knows about because the slap and kill during halos will always be the best and most used tool they have. The only good way to make the others more central is to remove the slap and kill.
> > > 
> > > 
> > > maybe if you want to try and force the meta to change, you gotta buff the other toys cult have, I have tried using the lesser known toys cult uses and it is shit, it's not just the funny cult hand being op, the powercreep of everything else has made any cult stuff except for constructs and red hand unviable
> > > cult is pretty weak already and is only held up by the funny stun hand. try buffing them instead of just making the cult weak so that the meta can change
> > 
> > 
> > "weak" infinite printable eswords with the strongest flat armor in the game space proof with magic teleport spells as a conversion antag with built in emps and revival. This antag is literally every antag mixed into a grab bag of some of their strongest abilities.
> 
> The thing about all of these abilities that Cult has is that they rely on one quality that 90% of people don't grasp, which utterly hamstrings the cult:
> 
> Cooperation and communication. Do you have any idea how many times I've watched a cult fumble on a summon rune because a significant chunk of the cult is too busy trying to murderbone and somehow can't see the GIANT RED TEXT from cult chat yelling "GO TO X NOW GO TO X NOW GO GO GO GO NOW NOW NOW X X NOW GO X NOW GO"?
> 
> Or how many times I've seen someone who actually gets runes trying to get anyone to stand next to it long enough to activate it? I don't think I've seen anyone pop a blood boil in years, it's sad. It's no wonder POCKET SHADE was such a meta strat for the longest time, it was the only way to get some of these clueless fuckwits to actually stand near a rune when you needed them.

The funniest part is that if the wiki isn't lying bloodboil would crit anyone in sight but people still don't do it

Summoning even when the rune is done is extremely difficult because people don't know how to lay down and stand on the rune and because every cultist's ego is too big to vote someone else as leader just so they can use the teleport spell and summon.

**Comment by san7890** [comment]
i don't forsee this getting merged and i think the arguments in favor of this feature weren't helpful or were too hostile (also the branch name "god-i-still-hate-this-antagonist" doesn't really make me sympathetic)

we can something similar to this later but i think there should be more feedback telemetry on the ways cult magic is used so we can audit that thorougly. if anyone really feels strongly about this i encourage you to implement blackbox logging on this stuff into the game and let it run for a month then beg someone to release the SQL results to you as a csv file or somethin

**Pull request closed without merging**